Automattic/wp-calypso
# Odyssey Stats: Add a dismissable nudge for customer feedback after three views of Odyssey Stats

Related to #72927 

## Proposed Changes

* Adds a dismissible notice box above the Traffic page highlight section. 
* Notice box displays after 3 views of Odyssey Stats
* Notice box nudges user for customer feedback
* Refactored notices to StatsNotices component
* Refactored backend from JITM to Stat's own

Goal

![image](https://user-images.githubusercontent.com/30754158/217621297-8316d79a-f0ef-4d23-8645-66c6178b6c4f.png)

## Testing Instructions

- Setup Odyssey with Option 1 in `PejTkB-3E-p2` <- enable New Stats if not yet enabled
- Open `/wp-admin/admin.php?page=stats`
- Ensure you see the opt out notice

<img width="853" alt="image" src="https://user-images.githubusercontent.com/1425433/218000427-091526b7-afcd-4fde-940a-86a4cc1aa546.png">

- Refresh 3 times
- Ensure you see the new user survey notice

<img width="856" alt="image" src="https://user-images.githubusercontent.com/1425433/218000562-0a09638f-74b8-420b-ac9d-d99da2f9fdab.png">

- Ensure the 'x', 'Take quick survey' and 'Remind me later all work' <- you'll have to remove `stats_options` in `wp_options` to restart the flow to see it or you could hack the code a little for testing
- Refresh
- Ensure these notices don't come up again
		

## Pre-merge Checklist
